session18/src/practica/Main.java

```diff
@@ -0,0 +1,10 @@
+package practica;
+
+import java.util.List;
+
+public class Main {
+
+    public static void main(String[] args) {
+
```

Comment:
you could create some instances of the created classes from this package and call their methods here in order to play with them instead of leaving this empty `main` here.